src/helperFunctions.ts

```diff
@@ -238,3 +238,20 @@ export function getBotInviteUrl() {
     user().id
   }&permissions=3212353&scope=bot%20applications.commands`;
 }
+
+export enum TimestampType {
```

Comment:
Just found this definition in the discord.js docs, does this mean we could use a type from there instead of creating a custom one? if so, I'd prefer that

https://discord.js.org/#/docs/discord.js/main/typedef/TimestampStylesString

Reply:
I mean depends. Enums are highly controversial. So entirely depends on if you rather work with an enum or a type

Reply:
I like enums 😬

But if there's a type we don't need to maintain ourselves, that's the better option IMO

Reply:
Let's use the enum and see where it gets us 😬